### ISSUE TITLE
Request an update

### ISSUE THREAD

**Issue by zglei** [a2569875/lora-prompt-tool#19]
With the update of the SD version, it now appears that the json file obtained cannot support the configuration file from Civil. After research, it is the problem of the json file obtained. Please take a look at this bug. Thank you.

**Comment by Miuna88** [comment]
Yup, this has been a problem for a while, I posted about it back in October but this tool hasn't seen any updates in a while so it likely wont be patched, shame too as its by far the most useful prompt tool, much more useful than the other ones i have tried.
